activiti-core/activiti-spring-process-extensions/src/main/java/org/activiti/spring/process/ProcessExtensionService.java

```diff
@@ -46,4 +47,11 @@ public Extension getExtensionsFor(@NonNull ProcessDefinition processDefinition)
     public Extension getExtensionsForId(@NonNull String processDefinitionId) {
         return processExtensionRepository.getExtensionsForId(processDefinitionId).orElse(EMPTY_EXTENSION);
     }
+    public boolean hasEphemeralVariable(@NonNull String processDefinitionId,
+                                        @NonNull String variableName) {
+        Extension extension = this.getExtensionsForId(processDefinitionId);
+        VariableDefinition extensionPropertyByName = (extension != null) ? extension.getPropertyByName(variableName) : null;
+        return extensionPropertyByName!=null && extensionPropertyByName.isEphemeral();
```

Comment:
Can you remove all these null checks and refactor the snippet to use `Optional.ofNullable().map().ifPresent()`?